index.js

```diff
@@ -186,8 +186,18 @@ exports.viewAccount = async function (options) {
     if (state && state.amount) {
         state['formattedAmount'] = utils.format.formatNearAmount(state.amount);
     }
-    console.log(`Account ${options.accountId}`);
-    console.log(inspectResponse.formatResponse(state));
+    const storageInM=state.storage_usage/1e+6;
+    const accountDetails={
+        'Account:':`${options.accountId} at block ${state.block_height} (https://explorer.testnet.near.org/blocks/${state.block_hash})`,
+        [chalk.bold('amount:')]:`${state.amount} ${chalk.bold('yn')}`,
+        'locked:':state.locked,
+        'storage:':`${storageInM}M (${state.storage_usage} bytes)`,
```

Comment:
I would say that the storage field should be "smart" aswell. Storage in bytes is also required.

Reply:
Like this :
```
**Storage:**0.032597M
**Raw Storage:**32597 bytes
```
?

Reply:
@Rishabhraghwendra18 no, if it's less than 1 Megabyte it should be shown in Kilobytes.